DepotDownloader/ContentDownloader.cs

```diff
@@ -401,8 +398,12 @@ public static async Task DownloadAppAsync( uint appId, uint depotId, string bran
             KeyValue depots = GetSteam3AppSection( appId, EAppInfoSection.Depots );
 
 
-            if ( forceDepot )
+            if ( isUgc )
             {
+                var workshopDepot = depots["workshopdepot"].AsUnsignedInteger();
+                if (workshopDepot != 0)
+                    depotId = workshopDepot;

```

Comment:
Should this have a fallback to the appid?

Reply:
Yes. It's not obvious, but it does. The app id gets prepopulated in it here, https://github.com/SteamRE/DepotDownloader/pull/45/files#diff-0b2dbef7fd518e60935db1741d7490adR41